platform/src/api/v1/surveys/mod.rs

```diff
@@ -62,36 +62,51 @@ pub async fn get_survey() -> Result<GetSurveyResponse, ServerFnError> {
         crate::utils::logger::new_api("GET", &format!("/v1/email/{}/surveys/{}", email, survey_id));
     let cli = crate::utils::db::get(&log);
 
-    let res: Result<Option<GetSurveyResponse>, easy_dynamodb::error::DynamoException> = cli
-        .get(format!("{}#survey-detail#{}", email, survey_id).as_str())
+    let res_summary: Result<Option<SurveySummary>, easy_dynamodb::error::DynamoException> = cli
+        .get(format!("{}#survey#{}", email, survey_id).as_str())
         .await;
 
-    match res {
+    let res_question: Result<
+        (Option<Vec<Question>>, Option<String>),
+        easy_dynamodb::error::DynamoException,
+    > = cli
+        .find(
+            "gsi1-index",
+            None,
+            Some(100),
+            vec![("gsi1", format!("{}#survey-question#{}", email, survey_id))],
+        )
+        .await;
```

Comment:
아래와 같은 패턴을 조금더 권장합니다.
```suggestion
    let res_question: (Option<Vec<Question>>, Option<String>)   = cli
        .find(
            "gsi1-index",
            None,
            Some(100),
            vec![("gsi1", format!("{}#survey-question#{}", email, survey_id))],
        )
        .await?;
```

Reply:
해당 방식으로 수정해주었습니다!